docs/guide/adapters/README.md

```diff
@@ -175,6 +175,20 @@ _(in order of first appearance)_
   
     ![](../../images/ZigStar-PoE.png)
     </details>
+    
+  * <details>
+    <summary>SMLIGHT SLZB-06 ZIGBEE ETHERNET USB POE WIFI LAN ADAPTER</summary>
+  
+    Powerfull, tiny modern design, developed for zigbee2mqtt, PoE supported, pre-flashed and ready to use Zigbee adapter. Supports Zigbee 3.0, Ethernet, LAN, USB and WiFi connections. Rich packing that includes SLZB-06 adapter, +5dB antenna, screws, adhesive tapes, scredriver, microUSB-typeC adapter, screw-fix-helper and QR-manual. Distant update, modern UI open-source firmware. Optoelectronic isolation for Ethernet, LEDs and physical button for interacting with user. Fits Home and Business use.
+    * [Zigbee CC2652P firmware](https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator/Z-Stack_3.x.0/bin)  
+    * [ESP32 firmware](https://github.com/smlight-dev/slzb-06-firmware/)  
```

Comment:
- I see both ESP32 and CC2652P, does the device come with different chips?
- What CC2652P firmware does it use? launchpad or other?

Reply:
@Koenkk the device comes with both chips. CC2652P is a Zigbee chip; it handles Zigbee network.
ESP32 is a "peripheral" chip, it manages everything other within the device - as intermediary between CC2652P and Ethernet, as controller of LED, button etc.
Uses "other" firmware

Reply:
I should read better next time; I read EZSP and not ESP32 😆